README.md

```diff
@@ -1,17 +1,19 @@
 # Continous Deployment
 
-This repository contains an opinionated reference architecture to setup, manage and operate a continous deployment pipeline.
+[This repository](https://github.com/operate-first/continuous-deployment) contains an opinionated reference architecture to setup, manage and operate a continous deployment pipeline.
 
 ### Prerequisites
-Kustomize 3.8.1+
-SOPS 3.4.0+
-KSOPS 2.1.2+
+* Kustomize 3.8.1+
+* SOPS 3.4.0+
+* KSOPS 2.1.2+  
 
-Ensure you have the key to decrypt secrets. Reach out to members of the Data Hub team for access.
+Find more information on how to install the pre-requisites [here](docs/manage_your_app_secrets.md).
+
+Ensure you have the key to decrypt secrets. Open an issue in this [repository](https://github.com/operate-first/continuous-deployment/issues) requesting for access.
```

Comment:
`requesting for access.` -> `requesting access.`

Reply:
Fixed above and fixed whitespace errors